src/CMakeLists.txt

```diff
@@ -153,10 +153,11 @@ function(add_runtime_modules TARGET_RUNTIME_FILES TARGET_RUNTIME_DIR)
       set(BC "${PROJECT_BINARY_DIR}/${PROJECT_NAME}.build/${NATIVE_INT_DIR}initmod.${i}_${j}.bc")
       set(INITMOD_D "${INITMOD_PREFIX}${i}_${j}_debug.cpp")
       set(INITMOD "${INITMOD_PREFIX}${i}_${j}.cpp")
+      set(INCLUDE_DIRS -I${TARGET_RUNTIME_DIR})
```

Comment:
What happens if the TARGET_RUNTIME_DIR has a space in it? This probably needs quoting at both the definition and usage.